bip-0021.mediawiki

```diff
@@ -11,6 +11,12 @@
   Created: 2012-01-29
 </pre>
 
+=Supersceded by BIP 321=
+
+This BIP has been supersceded and replaced with BIP 321. Please see [[bip-0321.mediawiki|BIP 321]] instead.
```

Comment:
spelling nit for this line and the header: Should be "superseded", not "supersceded".